### DISCUSSION TITLE
Ansilble

### DISCUSSION THREAD

**Discussion by palsachennaiah** [learndevopsonline/learndevopsonline#1589] [Q&A] [answered]
![Screenshot 2023-11-09 073841](https://github.com/learndevopsonline/learndevopsonline/assets/148023299/4146c2af-7e90-4dc9-8974-d66d57d30739)
![Screenshot 2023-11-09 074036](https://github.com/learndevopsonline/learndevopsonline/assets/148023299/5eef7a6f-8226-4225-89ed-dcbe7fd05a06)
what is the solution here

**Comment by sreedharm07** [comment, marked as answer]
Hi @palsachennaiah 
give space after -name ( - name)
and 
give space after name:sample ( name: sample )
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/b9e2fe53-f02e-4d88-b6b5-806f6820a812)

**Comment by palsachennaiah** [comment]
Thank you.
